web-ui/src/pages/AccessCodePage.jsx

```diff
@@ -0,0 +1,37 @@
+import react from "react";
```

Comment:
This should be:
`import React from "react";`

The lowercase is making it fail the gradle build.